backend/prompt_studio/prompt_studio_output_manager_v2/serializers.py

```diff
@@ -25,4 +26,11 @@ def to_representation(self, instance):
             )
             token_usage = {}
         data["token_usage"] = token_usage
+        # Convert string to list
+        try:
+            context = json.loads(data["context"])
```

Comment:
@Deepak-Kesavan are we doing this to not change the model / serializer and run a migration?

Reply:
@chandrasekharan-zipstack We always use text to store data in postgres. We need to load the data every time using json.loads() before using. But in this case, we directly fetch from DB and return it to frontend. So instead of parsing it in the frontend, I handled it in the serializer.

Reply:
@Deepak-Kesavan if we know that the data will always be JSON we can attempt to alter the field data type in postgres rather than doing an extra serialization / deserialization  while storing in the DB

Reply:
It will be an unwanted effort to migrate table schema and do data migrarion on existing data. Even after doing that the chunk data will be still wrong if it is not ran again